### ISSUE TITLE
Rollbar $last_error['type'] only support E_ERROR

### ISSUE THREAD

**Issue by diogopms** [rollbar/rollbar-php#55]
Hi, 
I needed to change this method, because my system (nginx+php5-fpm) i have  $last_error['type'] = 4 instead of 1 (E_ERROR).

Can you give help me or give information about that, because i want to use rollbar if some syntax error appear.

```php
public static function report_fatal_error() {
        // Catch any fatal errors that are causing the shutdown
        $last_error = error_get_last();

       
        if (!is_null($last_error)) {
             $last_error['type'] = E_ERROR;
            switch ($last_error['type']) {
                case E_ERROR:
                    self::$instance->report_php_error($last_error['type'], $last_error['message'], $last_error['file'], $last_error['line']);
                    break;
            }
        }
    }
```

**Comment by coryvirok** [comment]
Looks like we should add in a new `case`:

```php
switch ($last_error['type']) {
    case E_PARSE:
    case E_ERROR:
        self::$instance->report_php_error($last_error['type'], $last_error['message'], $last_error['file'], $last_error['line']);
        break;
}
```

@brianr look good?

**Comment by brianr** [comment]
Looks good to me.

**Comment by brianr** [comment]
Fixed in #56

**Comment by brianr** [comment]
Fix released in version 0.11.2
